lib/phoenix/channel/transport.ex

```diff
@@ -206,4 +206,40 @@ defmodule Phoenix.Channel.Transport do
     end
     :ok
   end
+
+  @doc """
```

Comment:
Doc missing. :) Also, can we unit test this function directly instead of testing all combinations through websockets and long polling? The unit tests will be much, much faster.